src/translators/sql_translator.py

```diff
@@ -397,6 +397,7 @@ def _insert_entity_rows(self, table_name: str, col_names: List[str],
             )
             self._insert_original_entities_in_failed_batch(
                 table_name, entities, e)
+            raise
```

Comment:
introducing this, two translator tests fails:
https://app.circleci.com/pipelines/github/orchestracities/ngsi-timeseries-api/352/workflows/2116016d-0161-40ff-84d8-e6b17089c6d8/jobs/1410/tests#failed-test-1

https://app.circleci.com/pipelines/github/orchestracities/ngsi-timeseries-api/352/workflows/2116016d-0161-40ff-84d8-e6b17089c6d8/jobs/1410/tests#failed-test-3

Reply:
for now let's comment it out, but let's talk about the logic here, and let's agree what should happen.

Reply:
I don't think that `raise` statement should be there at all. should we zap it?

when the flow gets to that point, the outcome should be determined by the call to `_insert_original_entities_in_failed_batch`.